frontend/src/components/HistoryTable.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ReactTable from 'react-table';
+import 'react-table/react-table.css';
+import { STATUS_LOADING, STATUS_ERROR, STATUS_SUCCESS } from '../state/query';
+
+class HistoryTable extends Component {
+  render() {
+    const { items, onOpenQuery } = this.props;
+    const columns = [
+      { Header: 'query', accessor: 'sql' },
+      {
+        Header: 'log',
+        id: 'log',
+        accessor: row => {
+          switch (row.status) {
+            case STATUS_SUCCESS:
+              return 'Query runned in Nsec';
+            case STATUS_ERROR:
+              return `Query failed - ${row.errorMsg}`;
+            case STATUS_LOADING:
+              return 'Query is running';
+            default:
+              return 'unknown status';
+          }
+        }
+      },
+      {
+        Header: 'date',
+        accessor: 'datetime',
+        Cell: ({ value }) => value.toString()
+      },
+      {
+        Header: 'action',
+        accessor: 'sql',
+        Cell: ({ value }) => (
+          <a onClick={() => onOpenQuery(value)}>open query</a>
```

Comment:
For some reason I don't get a pointer cursor on this link.

Reply:
That's correct. The link without `href` doesn't have pointer cursor. But before you were strongly against `href='#'`. Maybe I need to add addition css rule.